aws/resource_aws_default_route_table.go

```diff
@@ -93,6 +93,11 @@ func resourceAwsDefaultRouteTable() *schema.Resource {
 							Optional: true,
 						},
 
+						"vpc_endpoint_id": {
```

Comment:
You might add an issue to track the cross resource read if it's something that needs fixing.

Reply:
Created https://github.com/hashicorp/terraform-provider-aws/issues/16145